packages/firebase_ml_vision/ios/Classes/LocalVisionEdgeDetector.m

```diff
@@ -0,0 +1,82 @@
+#import "FirebaseMlVisionPlugin.h"
```

Comment:
Just like the Java versions, you can probably share alot of the logic with the `ImageLabel.m`.